11-multi-track/bb_views_part1/readme.md

```diff
@@ -111,14 +136,15 @@ Just like in vanilla JS, we usually want to render HTML and put it into the
 `el`. This method could be called anything, but convention is to call it
 `render`.
 
-### Exercise: Define a basic render method (10 minutes)
+### Exercise: Define a Basic Render Method (10 minutes)
 
 Use the `.html()` method to fill your view's `el` with some basic HTML. For now,
 just make an `<h2>` and set it's content to be the grumble's title.
+> Not sure how to begin? Think about how we defined `render` in our [earlier OOJS classes](https://github.com/ga-dc/curriculum/tree/master/08-single-page-web-apps/oojs1-read#views-artist-2090). Your Backbone render method in this exercise, however, will be much shorter.
 
 Remember that each view has a `model` property.
 
-Test your view by instantiating one, calling `.render()`, and looking at the el:
+Test your view by instantiating one, calling `.render()`, and looking at the `el`...
```

Comment:
might be good to have a defined render function here, or in some cliffnotes at the end.